Tests/SwiftParserTest/translated/DollarIdentifierTests.swift

```diff
@@ -57,7 +57,8 @@ final class DollarIdentifierTests: XCTestCase {
       """,
       diagnostics: [
         // TODO: Old parser expected error on line 2: '$' is not an identifier; use backticks to escape it, Fix-It replacements: 9 - 10 = '`$`'
-        DiagnosticSpec(message: "expected identifier and member block in class"),
+        DiagnosticSpec(message: "expected identifier in class"),
+        DiagnosticSpec(message: "unexpected code '$' in class"),
```

Comment:
I suppose we should add `dollarIdentifier` to `expectIdentifier`? I can do that in another PR. I probably shouldn't have added the start of line check in this one either.

Reply:
I think/hope this should be fixed by https://github.com/apple/swift-syntax/pull/998.

Reply:
Ah thank you. I'll rebase 🙇